src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java

```diff
@@ -1467,4 +1484,39 @@ private List<Map<String, Object>> sortDataByCriteria(List<Map<String, Object>> d
 
 		return data;
 	}
+
+	private Map<String, Object> isWFRequestExist(WfRequest wfRequest) throws IOException {
+
+		if (wfRequest == null || wfRequest.getUpdateFieldValues() == null || wfRequest.getUpdateFieldValues().isEmpty()) {
+			throw new IllegalArgumentException("Invalid WfRequest or updateFieldValues");
+		}
+
+		Map<String, Object> response = new HashMap<>();
+		String newRequestKey = ((Map<String, Object>) wfRequest.getUpdateFieldValues().get(0).get(Constants.TO_VALUE)).keySet().iterator().next();
```

Comment:
Is validation applied for wfstatus otherwise the get after get(0) will cause the null pointer exception